CIP-Draft-Initial-Params/Initial.md

```diff
@@ -0,0 +1,172 @@
+## Preamble
+
+---
+* CIP: ?
+* Title: Cardano Protocol Parameters
+* Authors: Kevin Hammond (<kevin.hammond@iohk.io>)
+* Discussions-To: <kevin.hammond@iohk.io>
+* Comments-Summary: <summary tone>
+* Comments-URI: <links to wiki page for comments>
+* Status: Draft
+* Type: Standards Track
```

Comment:
```suggestion
* Type: Informational
```
Given that this is the initial Shelley params, this isn't something that can be changed itself (these really are what the initial Shelley params were, the past is immutable!), only superseded by later updates.

We've concluded that it makes most sense for param changes to be separate follow-up CIPs.

Reply:
Agreed.